lib/local_notifications.dart

```diff
@@ -135,7 +135,7 @@ class LocalNotifications {
       launchesApps,
       presentWhileAppOpen
     ];
-    return await _channel.invokeMethod(_createNotification, args);
+    await _channel.invokeMethod(_createNotification, args);
```

Comment:
Any reason for this change?

Reply:
Basically the id is passed in as parameter, so we don't need to return it :P

Reply:
It's a design decision, but I think it might confuse some people, because in most cases you should/must pass an id as parameter to `createNotification`

Reply:
This is a good idea. I think it makes sense to require the id to be passed in as parameter, and have this return value be null. I won't block merging this PR waiting for these changes, but this is something I'd like to have before pushing out a new version of the plugin.